apis/v1alpha1/tlsconfig_types.go

```diff
@@ -65,6 +65,14 @@ type TLSConfig struct {
 	//
 	// +required
 	CertificateRef CertificateObjectReference `json:"certificateRef,omitempty" protobuf:"bytes,2,opt,name=certificateRef"`
+
+	// AllowedInRoute dictates if TLS certificate can be configured via Routes
```

Comment:
Nit: I think this should either by "if a TLS certificate" or "if TLS certificates"